docs/installation/opni_agent/enable_tracing.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Enable Traces Manually in downstream clusters
+slug: /installation/opni_agent/enable_tracing
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+Opni agent doesn't support the tracing capability yet. This guild will walk you through an example of using Opentelemetry Collector to ship trace data to Opni.
+
+## Prerequisites
+* The [Logging Backend](/docs/installation/opni/backends.md) is enabled in your Opni cluster.
+* Instrument your applications in the downstream cluster.
+* Cert-manager is installed in the downstream cluster.
+    <summary>Install cert-manager using kubectl apply with static manifests</summary>
+
+    ```bash
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
+    ```
+
+## Getting Started
+**1. Install Data-Prepper**
+:::note
+if you already have Opni-Agent installed in your downstream cluster and have the logging capability enabled, skip this step.
+In this case, 
```

Comment:
unfinished sentence?

Reply:
let's assume the user has enabled the logging capability. we will support traces without logs in the future but for now let's force logs+traces to be enabled at the same time

Reply:
ok.